plugin/teksi_wastewater/interlis/utils/various.py

```diff
@@ -52,7 +53,8 @@ class CmdException(BaseException):
 
 
 def exec_(command, check=True, output_content=False):
-    logger.info(f"EXECUTING: {command}")
+    command_masked_pwd = re.sub(r"(--dbpwd)\s\"[\w\.*#?!@$%^&-]+\"", r'\1 "[PASSWORD]"', command)
```

Comment:
What about this to be safe?
```suggestion
    command_masked_pwd = re.sub(r"(--dbpwd)\s\".+\"", r'\1 "[PASSWORD]"', command)
```

Reply:
and is the password is always in double quotes?
i.e. is there a risk of $ sign substitution if it's in double rather than single quotes? (which would an issue on its own)